src/openzaak/components/catalogi/tests/admin/test_resultaattype_admin.py

```diff
@@ -589,3 +589,34 @@ def test_create_resultaattype_for_published_zaaktype_not_allowed(self, m):
             },
         )
         self.assertEqual(ResultaatType.objects.count(), 0)
+
+    def test_resultaattype_detail_with_read_only_user_invalid_domain(self, m):
```

Comment:
I tend to tag regression tests with the github issue number, that way you can easily run them with `src/manage.py test openzaak --tag gh-1962` for example
```suggestion
    @tag("gh-1962")
    def test_resultaattype_detail_with_read_only_user_invalid_domain(self, m):
```